donate/payments/tests/test_views.py

```diff
@@ -489,34 +492,6 @@ def test_get_success_url(self):
             reverse('payments:newsletter_signup')
         )
 
-    def test_get_transaction_details_for_session(self):
```

Comment:
Monthly does not result in a transaction, instead it yields a subscription, with the actual transaction getting triggered via web hook. As such we removed this test.